kpi/paginators.py

```diff
@@ -31,6 +31,19 @@ def get_parent_url(self, obj):
         return reverse_lazy('api-root', request=self.context.get('request'))
 
 
+class FastAssetPagination(Paginated):
```

Comment:
I would have given a name related to ProjectView (e.g. `ProjectViewAssetPagination` 😅 ) but if you think it can be reused somewhere else and you want to keep it as-is, please move it below `AssetPagination` to keep alphabetically sort 🙏 .

Reply:
I'm hoping it's reusable, so I'll keep the name and just move it. BTW the rest of the file is definitely not in alphabetical order